internal/godoc/render.go

```diff
@@ -151,6 +152,10 @@ func (p *Package) renderOptions(innerPath string, sourceInfo *source.Info, modIn
 		if p.Line == 0 { // invalid Position
 			return ""
 		}
+		// resolve gop_autogen.gop path
+		if strings.Contains(p.Filename, ".gop") || strings.Contains(p.Filename, ".gox") {
```

Comment:
don't need check file ext

Reply:
> don't need check file ext

I don't know if my comment caused the misunderstanding.

If not, what I am parsing here is this line of code
`//line web/filter/details/order.gop:15:1`

If the generated go file is jumped, there will be an extra prefix of the file name.

Reply:
I seem to understand. Lao Xu, what you mean is that you can delete the prefix directly and ignore the extension.

Reply:
yes.